LION/experiments/ct_benchmarking_experiments.py

```diff
@@ -171,12 +136,11 @@ def default_parameters(dataset="2DeteCT"):
         param.data_loader_params.task = "sino2recon"
         param.data_loader_params.input_mode = "mode2"
         param.data_loader_params.target_mode = "mode2"
-        
+
         return param
 
 
 class LimitedAngle60CTRecon(Experiment):
-
     def __init__(self, experiment_params=None, dataset="2DeteCT", datafolder=None):
         super().__init__(experiment_params, dataset, datafolder)
 
```

Comment:
Below there should then also be 
param.name = (
            "60 degrees Limited Angle CT reconstruction experiment from mode 2"
        )